Sources/Controllers/DashboardOnMap/MapSettings/MapSettingsGpxViewController.swift

```diff
@@ -862,6 +861,16 @@ extension MapSettingsGpxViewController: UISearchBarDelegate {
     }
 }
 
+extension MapSettingsGpxViewController: UISearchControllerDelegate {
+    func presentSearchController(_ searchController: UISearchController) {
+        DispatchQueue.main.async {
```

Comment:
Do we really need to use DispatchQueue.main.async here?

Reply:
Yes